recipes/nmslib/all/conanfile.py

```diff
@@ -25,10 +25,15 @@ class Nmslib(ConanFile):
     options = {
         "shared": [True, False],
         "fPIC": [True, False],
+        "build_extras": [True, False],
     }
     default_options = {
         "shared": False,
         "fPIC": True,
+        "build_extras": True,
```

Comment:
I'd use `with_extras` instead, what do you think?
```suggestion
        "with_extras": True,
```

Reply:
Oh, that makes sense. Agree 😄